### DISCUSSION TITLE
Static Media Failure

### DISCUSSION THREAD

**Discussion by buzzlightyear2k** [netbox-community/netbox#9917] [Q&A]
Im trying to install Netbox and encounter thi error.Can someone help me please !!! 
![image](https://user-images.githubusercontent.com/86591649/182759980-b778f531-4adf-40b2-a9d6-4f3599f3e3b3.png)

**Comment by donbrownmscs** [comment]
Don't try and navigate to to https://localhost:8001 in your web browser. The basic Netbox installation configures the web server (nginx or apache2) to listen on TCP port 443 (https). So, we want to navigate to https://localhost to see the Netbox home page.

Now, in Ubuntu, when we install apache2 or nginx, it creates the default directories (/var/www/html) and puts a default index.html file in there so we can see that Apache (or nginx) is working. 

So, in order to see the Netbox home page, we need to delete or rename the /var/www/html/index.html file.
To delete it:
$ sudo rm /var/www/html/index.html
To rename it:
$ sudo mv /var/www/html/index.html /var/www/html/index.bak
